src/biotite/database/uniprot/download.py

```diff
@@ -0,0 +1,190 @@
+# This source code is part of the Biotite package and is distributed
+# under the 3-Clause BSD License. Please see 'LICENSE.rst' for further
+# information.
+
+__name__ = "biotite.database.uniprot"
+__author__ = "Maximilian Greil"
+__all__ = ["_get_database_name", "fetch", "_assert_valid_file"]
+
+from os.path import isdir, isfile, join, getsize
+import os
+import io
+import requests
+from ..error import RequestError
+
+fetch_url = "https://www.uniprot.org/"
+
+
+def _get_database_name(uid):
```

Comment:
I think in *UniProt* the IDs are not called *UIDs* in contrast to the NCBI databases.